Firestore/Example/Podfile

```diff
@@ -39,6 +39,7 @@ target 'Firestore_Example_iOS' do
   pod 'Firebase/CoreOnly', '6.6.0'
 
   pod 'FirebaseFirestore', :path => '../../'
+  pod 'FirebaseCore', :path => '../../'
 
```

Comment:
I suggest adding GoogleUtilities back as well.

Reply:
#3648 effectively does that for CI purposes.